resources/import.php

```diff
@@ -305,28 +309,31 @@ function searchForShortName($shortName, $array)
 							}
 						}
 					}
-					foreach ($allIsbnOrIssn_columns as $columnCriterion)
+					if (!empty($allIsbnOrIssn_columns))
 					{
-						if($columnCriterion['delimiter'] !== '')
+						foreach ($allIsbnOrIssn_columns as $columnCriterion)
 						{
-							$columnValues = explode($columnCriterion['delimiter'],$data[$columnCriterion['column']]);
-							foreach($columnValues as $value)
+							if($columnCriterion['delimiter'] !== '')
 							{
-								if($value != '')
+								$columnValues = explode($columnCriterion['delimiter'],$data[$columnCriterion['column']]);
+								foreach($columnValues as $value)
 								{
-									$isbnIssn_values[] = $value;
+									if($value != '')
+									{
+										$isbnIssn_values[] = $value;
+									}
 								}
 							}
-						}
-						else
-						{
-							if($data[$columnCriterion['column']] != '')
+							else
 							{
-								$isbnIssn_values[] = $data[$columnCriterion['column']];
+								if($data[$columnCriterion['column']] != '')
+								{
+									$isbnIssn_values[] = $data[$columnCriterion['column']];
+								}
 							}
 						}
 					}
-					$deduping_count = count($resourceObj->getResourceByIsbnOrISSN($deduping_values));
+					$deduping_count = isset($deduping_values) ? count($resourceObj->getResourceByIsbnOrISSN($deduping_values)) : 0;
```

Comment:
This isn’t about that function, this is about the `$deduping_values` variable.

> PHP Notice:  Undefined variable: deduping_values in /var/www/html/coral/resources/import.php on line 336, referer: http://localhost/resources/import.php

The variable is not necessarily set in any of the code above this line, so I added the `isset()` to simply check if it had been and then count the resource objects or set it to 0.